recipes/py-linux-ports/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "py-linux-ports" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2d7e6ecfe4458c9c087d414013ba2961a6a0541c5d441eea8d432d456a31ce01
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
```

```suggestion
    - python >=2.7
```

Ref https://github.com/sujitmandal/py-linux-ports/blob/ca08fa238cc40ed0e808f2ee1f02be0d10404a7c/setup.py#L25

But you'll need to confirm if this is correct.